themes/finna2/js/finna-feed-tabs.js

```diff
@@ -78,6 +78,13 @@ finna.feedTabs = (function finnaFeedTab() {
         parent.setAttribute('aria-selected', false);
       }
     });
+    if (element.parentNode.nextElementSibling != null) {
```

Comment:
Tämä toimii kyllä accordionissa, mutta tabeilla tapahtuu erittäin mielenkiintoisia asioita. Turvallisempaa olisi manipuloida suoraan `_.tabContent`, kun se kerran on tiedossa. Ja samalla jättää initialisointi tekemättä, jos tabContent on piilotettu. Jotain tällaista ehkä:

    _.tabContent.innerHTML = '';
    _.tabContent.classList.add('hidden');
     delete _.tabContent.dataset.init;
    _.anchors.forEach(function removeActive(el) {
      var parent = el.parentNode;
      if (el.dataset.tab === tab && !parent.classList.contains('active')) {
        parent.classList.add('active');
        parent.setAttribute('aria-selected', true);
        if (el.classList.contains('feed-accordion-anchor')) {
          parent.insertAdjacentElement('afterend', _.tabContent);
        }
        _.tabContent.dataset.feed = tab;
        finna.feed.loadFeed(_.tabContent, function onLoad() {
          _.isLoading = false;
          if (!_.allowHashChange) {
            _.allowHashChange = true;
          }
        });
      _.tabContent.classList.remove('hidden');
      } else {
        parent.classList.remove('active');
        parent.setAttribute('aria-selected', false);
      }
    });

Siinä sitten vielä ongelmana, että tabien kanssa toiminnallisuus on outo. Tabia ei yleensä pysty sulkemaan klikkaamalla, vaikka yläoleva sen tekeekin.

Reply:
Testailin tuolla laittamallasi koodilla ja se toimii hyvin, niin korvasin sillä. Mietin itseäni käyttäjänä, niin en varmaan yrittäisi tabeja sulkea, koska tietokoneen näytöllä on riittävästi tilaa. Mutta mobiilissa sulkemisen mahdollisuus korostuu.